handbook/company/leadership.md

```diff
@@ -410,7 +410,36 @@ When it comes to performance feedback, [speak freely](https://fleetdm.com/handbo
+3. Non-managerial team members have low-level insight into their duties and responsibilities that may help Illuminate strategies that better assist themselves, their team, or other departments.
+4.  The CEO may discover insightful details regarding managers from a different perspective that may help them better support these managers.
+
+### The How-To
+
+**Actions For The Manager:**
+
+1.  Once communication from a team member has been received that they wish to initiate a Skip-Level 1:1 OR you would like to initiate a Skip-Level 1:1 on behalf of a Fleetie, you may do so by sending `@mikermcneil` a DM in Fleet Slack, at-mention our CEO in the [#help-leadership channel](https://fleetdm.slack.com/archives/C02HWSTJ17Z), or [schedule time with the CEO](https://fleetdm.com/handbook/company/communications#schedule-time-with-the-ceo).
+2.  In your communication to your direct report, be sure to link them to this section of the document so they have a clearer understanding of the Skip-Level 1:1 process, meeting structure, and helpful recommendations. 
+3.  After initiation is made with the CEO, managers should then make sure to help the CEO, and Fleetie establish communication to ensure a timeline has been set in motion for the two to meet. Managers then only need to check in with their direct reports to make sure everything is going accordingly.
+
+**Recommendations For The Non-Manager Fleetie:**
+
+1.  Take a breath first and read through the [CEO’s Flaws](#ceo-flaws). This will hopefully give you a better understanding of Mike and should hope to serve both you and Mike in making the conversation a smoother experience.
+2.  We highly recommend you read through the [Performance Feedback](#performance-feedback) section, the items listed here are a set of expectations that lay at the heart of all Fleet 1:1 meetings. That section should help you with the processing of feedback as well as presenting it.
+3.  Please use the template [here](https://docs.google.com/document/d/1AyZPGRdLtGXFU6TRndbTqs033rWSKy5dUvXU3d4LLZc/copy). The answering of the three questions on the template are pertinent to the success of the meeting, however, we would like to emphasis that if you have talking points beyond the items listed on the template, we empower you to share, and add them to the document. If you feel uncomfortable speaking about items on the template, we hope you can help the CEO identify why that is.
+4.  Once you have completed the document, please send the document to Mike via the communication pipeline you two have established. Turnaround time for these meetings will be communicated to you by Mike. The length of time for the meeting can scale, however, we do ask you to try to keep Mike’s schedule in mind and limit meetings to one hour or less. Never fear though, if you have items that weren’t fully completed in one of your meetings, just be sure to let Mike know that you have additional content to go over and would like to set up an additional meeting.
```

Comment:
Covered above 
Side note: finding related words to like to rather than "here" is always better for SEO

```suggestion  
```